index.js

```diff
@@ -15,6 +15,15 @@ module.exports = {
         separator: '+',
         filePattern: 'index.html',
         versionFile: 'package.json',
+
+        commitHashLength: function() {
+          if (this.type === 'git-commit') {
```

Comment:
This was added to ensure that we are not breaking the current behavior even though it is inconsistent.